### PR TITLE
Remove path to specific display HAL

### PR DESCRIPTION
**Pull request by stellirin**
Remove path to specific display HAL

Build will ALWAYS fail on msm8974 because QServiceUtils.h does not exist. In fact this camera HAL probably can never be used with unmodified AOSP display HAL due to missing parts.

Sadly Google does not maintain its devices nor its HALs to keep up-to-date with QCOM developments. Google simplay adds a new HAL for new devices. This means the msm8974 lacks the newer parts
found in the msm8994 HAL.

In other words the msm8974 is an old POS.

The correct fix is to export the QServiceUtils.h header with the other display headers.

https://android-review.googlesource.com/#/c/238530

We also need to update the msm8974 HALs or find a way for msm8974 devices to use the msm8994 HALs.

This reverts commit 18fc8d093872bccc8a4708e7fe7aa2d84b539f19.

### CODE REVIEW

**Comment by jerpelea** [comment]
or this commit can be rebased 
https://github.com/sonyxperiadev/camera/commit/0f2f305009d1cf55bde02116696d262ffdc84609
